TESTS/psa/its_ps/main.cpp

```diff
@@ -16,8 +16,8 @@
 * limitations under the License.
 */
 
-#ifndef TARGET_PSA
-#error [NOT_SUPPORTED] ITS/PS tests can run only on PSA-enabled targets.
+#if !defined(TARGET_PSA) || !defined(MBED_CONF_RTOS_PRESENT)
```

Comment:
as earlier mentioned, please split the macro qualifier checks, and give different error message separately